python/ball_launcher_beepy/ball_launcher_gui.py

```diff
@@ -0,0 +1,333 @@
+# ---------------------------------------------------------------
+import time
+
+from tkinter import *
+from tkinter import messagebox
+
+from RPi import GPIO
+from xOC05 import xOC05
+from xOC03 import xOC03
+
+
+# Initialisierung XinaBox OC 05

```

Comment:
It would be good to wrap the code in an `if __name__ == "__main__":`-block to avoid accidental execution in case the file gets imported somewhere for some reason.